docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,267 @@ include::includes/global-attributes.asciidoc[]
+In _Definition_ ::
+* _IP Address/MAC Address/Range (CIDR)_, give the network adress of your _Production_ network,for us, it will be *172.16.1.0/24*
+* _Description_, give a description so you can quickly see what it is about,
+* From the _Type_ list, select *AeroHIVE AP*.
 
-Configure the AAA Servers
-+++++++++++++++++++++++++
-On the left menu, click on 'Advanced Configuration', then 'Authentication', then 'AAA Client Settings'. We need to add the PacketFence RADIUS server here. Click on the 'New' button, and provide follow this configuration.
+NOTE: *TIP*: You can type directly the name you are looking forin the field
 
-image::docs/images/aerohive-radius-pf.png[scaledwidth="100%",alt="RADIUS server configuration"]
+* From _Mode_ select *Production*
+* _Switch Group_ by defaultset to *None*.
+* _Deauthentication Method_ set to *RADIUS*
+* Click *SAVE*
 
-Click Ok to save your changes.
+image::docs/images/pfaerohive10.png[scaledwidth="100%",alt="New switch configuration",align=right]
 
-Configure the Open SSID
-+++++++++++++++++++++++
-On the left menu, click on 'SSIDs'. Click the 'New' button, and configure the SSID like in the following screen capture:
+'''
 
-image::docs/images/aerohive-ssid-public.png[scaledwidth="100%",alt="SSID open configuration"]
 
-Click OK to save the SSID.
+In _Role_ ::
+* Set all VLans ID in each roles.
 
-Configure the Secure SSID
-+++++++++++++++++++++++++
-On the left menu, click on 'SSIDs'. Click the 'New' button, and configure the SSID like in the following screen capture:
+image::docs/images/pfaerohive11.png[scaledwidth="100%",alt="New switch configuration",align=right]
 
-image::docs/images/aerohive-ssid-secure.png[scaledwidth="100%",alt="SSID secure configuration"]
+'''
 
-Click OK to save the SSID.
 
-Configure User Profile
-^^^^^^^^^^^^^^^^^^^^^^
-You will now need to create user profile matching your VLAN assignments, the "Default VLAN" have been created before to match the desired VLAN number.
+In _RADIUS_ ::
+* Modify the secret passphrase previously sets in the AeroHive Hive Manager,
+* Click on *SAVE*
 
-Create user profile as follow:
+This end the PacketFence configuration.
 
-image::docs/images/aerohive-user-profile-reg.png[scaledwidth="100%",alt="User profile registration"]
+'''
 
-image::docs/images/aerohive-user-profile-prod.png[scaledwidth="100%",alt="User profile production"]
+Web Auth Configuration
+----------------------
 
-Configure SNMP
-^^^^^^^^^^^^^^
-You now need to configure the SNMP to add a valid read community string. Click on the 'Configure' pane, and on 'Advanced Configuration', then 'Management Policies', then 'SNMP Assignments'. Now click on 'New' and follow this configuration.
+In this part we will describe the WebAuth configuration for Aerohive APs.
 
-image::docs/images/aerohive-add-snmp.png[scaledwidth="100%",alt="SNMP configuration"]
 
-<<<
+Assumption :
+~~~~~~~~~~~~
+We will use the same previous contexte, but we will have only one VLAN, the Production (11)
```

Comment:
rephrase the "same previous contexte".

as for your "Production" VLAN, you should put it above in your assumptions.